_rules/iframe-accessible-name-cae760.md

```diff
@@ -75,6 +75,14 @@ This `iframe` element gets its [accessible name][] from the content of the `div`
 <iframe aria-labelledby="frame-title-helper" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
 ```
 
+#### Passed Example 4
+
+This `iframe` element gets its [accessible name][] from the `title` attribute. Note, that specifying a positive `tabindex` value only alters the order of the [sequential focus navigation][], and as such the contents of the `iframe` element can be accessed.
+
+```html
+<iframe title="Top 5 Marathon runners" src="/test-assets/SC4-1-2-frame-doc.html" tabindex="5"> </iframe>
```

Comment:
I know that 4.1.2 does **not** require the name to be descriptive 🙄 but shouldn't we nonetheless try and have descriptive names in our examples? Especially given that we use the exact same doc in the `iframe` but name it once "Top 5 Marathon runners" and another time "Highlights from the World Cup"…